### PR TITLE
Blog post for claw game app case study

### PR DESCRIPTION
**Pull request by HipsterBrown**
This blog post covers how my team at Viam used Stately Studio and XState to model and refactor our robotic claw game controls application.

### CODE REVIEW

File: blog/2024-07-01-claw-game-state-machine/index.mdx [hunk 1]
```diff
@@ -0,0 +1,494 @@
+---
+title: Stately Case Study - Smart Machines Meet State Machines
```
Comment (davidkpiano) [inline blog/2024-07-01-claw-game-state-machine/index.mdx:2]:
```suggestion
title: Viam x Stately: Smart Machines Meet State Machines
```

File: blog/2024-07-01-claw-game-state-machine/index.mdx [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+description: Controlling hardware in the real world requires confidence and safety, even when it's part of a game!
+tags:
+  [
+    xstate,
+    stately studio
+  ]
```
Comment (davidkpiano) [inline blog/2024-07-01-claw-game-state-machine/index.mdx:8]:
```suggestion
  [
    case study,
    study,
    viam,
    stately,
    state machines
  ]
```

**Comment by HipsterBrown** [comment]
@davidkpiano Should I update the date on this post now?

**Comment by davidkpiano** [comment]
> @davidkpiano Should I update the date on this post now?

Yes please

**Comment by HipsterBrown** [comment]
Done!

**Review by davidkpiano — APPROVED** [review]

**Pull request merged**